rcsb/utils/struct/Scop2ClassificationProvider.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, cachePath, useCache, **kwargs):
         self.__nD, self.__ntD, self.__pAD, self.__pBD, self.__pBRootD, self.__fD, self.__sfD, self.__sf2bD = self.__reload(useCache=self.__useCache, fmt=self.__fmt)
         #
         if not useCache and not self.testCache():
-            ok = self.__fetchFromBackup()
+            ok = self.__fetchFromBackup(self.__fmt)
```

Comment:
Rather than add the format here, I would lean towards changing the default value in `__fetchFromBackup` below to be "pickle", since the fallBack file will generally continue to be in pickle format moving forward. What do you think?

If not, then I would just suggest adding `fmt=` explicitly to the argument here.

Reply:
I think `__fetchFromBackup` should match `__reload ` since that's where the fallback file is ultimately coming from. That said, I do agree it's a bit strange that `__fmt` is hard-coded as "pickle" in the `__init__ ` but defaults to "json" in those methods. So I think we should either leave it like this (with `fmt=`) or make the default value change in `__reload` also.

Reply:
OK, I see your point. I'm guessing the reason it's like this is that he had written the code originally for JSON, then realized the files were too big and decided to switch to pickle, in which case creating a `self.__fmt` variable was the easier thing to do than changing it everywhere (basically the same dilemma we're facing now). So, I guess the easiest solution is to leave how it is (with `fmt=`), but the cleaner solution would be to set the defaults to pickle everywhere (and including `__getAssignmentFileName`?). Either sounds OK to me though. I'll let you make the call :)

Reply:
I went with the "all of the above" option.